### DISCUSSION TITLE
Add `min_brightness` and `max_brightness` to `light.turn_on` service

### DISCUSSION THREAD

**Discussion by tal** [home-assistant/architecture#528] [Entity Models]
I've found it very common to have automations that have conditions on the current brightness of the light. In order to simplify many actions like this I was hoping to add a few parameters to the `light.turn_on` service.

- `min_brightness`: If the brightness is over the given value do nothing. If it is under bring the light up to that brightness, respects all other parameters.
- `max_brightness`: same as above but will bring the light down to this setting iff it's currently above it.

I'm happy to make this contribution but I wanted to find out if it's something that would be accepted before I start putting the work in.

Thanks

**Comment by amelchio** [comment]
Not as readable but you can already handle this with a template. For example, this should be `min_brightness: 100`:
```
brightness: '{{ [100, state_attr("light.office", "brightness") or 0] | max }}'
```

**Reply by tal** [reply]
Ah, thanks. Unfortunately I'm doing a lot of this though other automations, and not ones using yaml. I especially wanted to be able to set min/max in scenes.

**Reply by frenck** [reply]
scene restore states, min/max is not a state but a property. So it should not be part of a scene (just like a transition is not part of a scene).

**Comment by frenck** [comment]
Oh hi 👋 

This proposal has been lingering around for a long time, and basically stale. Considering the support for this proposal has been low, I will go ahead and close this proposal for now.

If renewed support might boil up in the future, we can always consider reopening and reconsidering.

../Frenck
